assets/scss/style.scss

```diff
@@ -1,7 +1,11 @@
 // Copyright (C) 2018 The Trustees of Indiana University
 // SPDX-License-Identifier: BSD-3-Clause
 
-@import "../../node_modules/rivet-uits/sass/rivet";
+@import "../../node_modules/rivet-uits/sass/core/variables";
+@import "../../node_modules/rivet-uits/sass/core/tools";
+
+@import "../../node_modules/rivet-uits/sass/components/badges";
+@import "../../node_modules/rivet-uits/sass/components/modals";
```

Comment:
I'm almost positive that the `@extends` rules that are causing the need to import this partial are because of some CSS/markup that isn't used anymore. Let's get together real quick in-person and I can help out with this.

It has to do with the `components/rvtd-search.scss` partial. Those are actually old style for a custom search that the site used to use before we replaced it with Algolia docsearch. Just an FYI.